Modules/_winapi.c

```diff
@@ -1537,42 +1537,66 @@ _winapi_PeekNamedPipe_impl(PyObject *module, HANDLE handle, int size)
 /*[clinic input]
 _winapi.LCMapStringEx
 
-    locale: LPCWSTR
+    locale: unicode
```

Comment:
Why change it for `locale`?

Reply:
I just backported the solution from https://github.com/python/cpython/commit/927b5afee73218500a8fa80df86216cfdc24ef5a#diff-42415407f8d0ef2d42e29d13d979f633e3543770e62c3871e1101ad532d336a8R1518

So, 3.11 and 3.12 would match.

Reply:
I agree. We don't need to treat this argument any differently - it's only the `src` argument that requires special handling.